src/types/index.ts

```diff
@@ -60,13 +60,18 @@ export type ArticleListItemRecord = {
   }
 }
 
+/* PublishableItemType is any item that has publishedDate and status */
+export type PublishableItemType = {
+  publishedDate: string
+  status: 'Draft' | 'Published' | 'Archived'
+}
+
 /* ArticleRecord is the complete article used when viewing the single article page */
-export type ArticleRecord = {
+export type ArticleRecord = PublishableItemType & {
```

Comment:
I will say I do like typescript's ability to combine types like this, but let me know if  it's unclear or if there is a better name. Some of our types end in `Type` and some don't.